recipes/lapack/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "3.6.0" %}
+
+package:
+  name: lapack
+  version: {{ version }}
+
+source:
+  fn: lapack-{{ version }}.tgz
+  url: http://www.netlib.org/lapack/lapack-{{ version }}.tgz
+  sha1: 7e993de16d80d52b22b6093465eeb90c93c7a2e7
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cmake
+    - gcc  # [unix]
```

Comment:
Good question. Unfortunately, we haven't decided what to do with Fortran or OpenMP yet.

As LAPACK requires Fortran and we don't include a Fortran compiler yet, we need to use the packaged `gcc` in this instance.